core/src/settings.rs

```diff
@@ -282,6 +282,8 @@ pub struct PropagatedState {
     #[serde(default)]
     pub(crate) hide_throw_halting_player: bool,
     #[serde(default)]
+    pub(crate) jack_variation: bool,
```

Comment:
Due to the many ways that people play the J->2 variation - do you mind making this an enum so we can add more versions later?

```
BackToTwoSetting {
  #[default]
  Disabled,
  SingleJackToTwo,
  // AceToTwo,
}
```